portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,44 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <nav>
+      <label class="logo">KvngSnow's Portfolio</label>
+      <ul>
+          <li><a href="http://www.github.com/JalenJenkins">Github</a></li>
```

Comment:
Fix the indentation of these children - use 2 spaces from the opening `ul` above.